kuma/users/views.py

```diff
@@ -352,7 +352,7 @@ def user_detail(request, username):
     """
     detail_user = get_object_or_404(User, username=username)
 
-    if (detail_user.active_ban and not request.user.is_superuser):
+    if (detail_user.active_ban and not request.user.add_userban):
```

Comment:
Should be ``request.user.has_perm('users.add_userban')``, not ``request.user.add_userban``.